deploy/terraform/modules/anydb_node/infrastructure.tf

```
@@ -0,0 +1,94 @@
+/*-----------------------------------------------------------------------------8
+Load balancer front IP address range: .4 - .9
++--------------------------------------4--------------------------------------*/
+
+resource "azurerm_lb" "anydb" {
+  count               = local.enable_deployment ? 1 : 0
+  name                = format("db-%s-lb", local.anydb_sid)
```

```suggestion
  name                = format("%s_xdb-alb", upper(local.anydb_sid))
```
I know you will work on a naming convention, but this way it is aligned with current app server.